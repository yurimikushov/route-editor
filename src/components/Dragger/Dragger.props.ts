type DraggerProps = {
  className?: string
}

export default DraggerProps
