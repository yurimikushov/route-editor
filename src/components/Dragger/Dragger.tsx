import { FC } from 'react'
import cn from 'classnames'
import { ReactComponent as DraggerIcon } from 'icons/dragger.svg'
import DraggerProps from './Dragger.props'

const Dragger: FC<DraggerProps> = ({ className }) => {
  return (
    <DraggerIcon className={cn(className, 'text-gray-500 hover:text-black')} />
  )
}

export default Dragger
