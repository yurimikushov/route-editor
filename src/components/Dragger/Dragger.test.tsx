import renderer from 'react-test-renderer'
import Dragger from './Dragger'

it('should render dragger control', () => {
  const tree = renderer.create(<Dragger className='w-6 h-6' />).toJSON()
  expect(tree).toMatchSnapshot()
})
