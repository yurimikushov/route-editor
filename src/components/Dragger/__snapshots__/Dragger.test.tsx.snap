// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render dragger control 1`] = `
<svg
  className="w-6 h-6 text-gray-500 hover:text-black"
>
  dragger.svg
</svg>
`;
